packages/cli-plugin-scaffold-node-package/template/src/index.ts

```diff
@@ -3,4 +3,5 @@
  * Reference it from other packages via the following import statement:
  * import { PACKAGE_NAME_CAMEL_CASED } from "PACKAGE_NAME";
  */
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
This file is a part of the Node Package scaffold, and with the `webiny scaffold` tool, it is copied to the user's project 

The thing is, there is no value in having the `eslint-disable` above the function, because users will start writing code in it (or they will remove it and write a class / plugin / whatever), and the `eslint-disable` immediately becomes redundant.

If you want to avoid `eslint` errors here, either ignore the file or maybe even better, just return some value from the function, e.g. `return Math.random();` or something cool.

Reply:
returned Math.random() as suggested.